src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

don't use transition: all, write only neede properties
```suggestion
    transition: color 300ms;
```